recipe/build.sh

```diff
@@ -2,6 +2,13 @@
 if [[ $target_platform =~ linux.* ]] || [[ $target_platform == win-32 ]] || [[ $target_platform == win-64 ]] || [[ $target_platform == osx-64 ]]; then
   export DISABLE_AUTOBREW=1
   $R CMD INSTALL --build .
+
+  # Installs kernelspec with absolute path and R version specific kernel naming:
+  # $R -e 'JuniperKernel::installJuniper(prefix = Sys.getenv("PREFIX"))'
+
+  mkdir -p "$PREFIX/share/jupyter/kernels/juniper"
+  cp -v inst/extdata/logo*.png "$PREFIX/share/jupyter/kernels/juniper/"
+  cp -v "$RECIPE_DIR/kernel.json" "$PREFIX/share/jupyter/kernels/juniper/"
```

Comment:
@dbast Is `/share/jupyter/kernels/juniper/` removed after running `conda remove r-juniperkernel`? If not, one option would be to delete the directory with a `pre-unlink.sh` file:

https://conda.io/docs/user-guide/tasks/build-packages/link-scripts.html

Reply:
Thanks for the hint. Yes, it is removed when removing the package ... tested that, after seeing it in irkernel https://github.com/conda-forge/r-irkernel-feedstock/blob/master/recipe/build.sh#L10 ...

Reply:
@dbast Great. Thanks!

Unrelated to my previous point, but FYI I can't install r-juniperkernel. Not sure what the issue is. It says it's not available on Anaconda Cloud when I run `conda install -c conda-forge r-juniperkernel `. In contrast, I can install r-irkernel with no problem. If this doesn't affect you, then just ignore because hopefully it is just a temporary problem with Anaconda Cloud.

Reply:
@jdblischak The juniper kernel could only be recently added to conda-forge, because of the new compilers... That means for Linux you need to have a gcc7 conda-forge environment, as there is no non-gcc7 linux package... aka `conda install -c conda-forge/label/gcc7 r-juniperkernel` ... the linux build was skipped before feedstock creation and got activated as part of the bot gcc7 PR...  see also the labels here https://anaconda.org/conda-forge/r-juniperkernel/files ... it is not obvious in the meta.yaml that there is no non-gcc7 linux build... the proper statement would be `skip: True  # [c_compiler == 'toolchain_c']` + rerender...

Reply:
@dbast I understand now. You're saying that there is no linux-64 package built for the old compilers (currently labeled [main](https://anaconda.org/conda-forge/r-juniperkernel/files?channel=main)) because the recipe can't be built with the old compilers. Thanks for the explanation!